From 83f932299b9969a1823b085d2269db677362f897 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Tue, 14 Jul 2020 19:18:25 +0800
Subject: [PATCH 08/49] LF-1762-21 gpu: drm: bridge: cadence: hdmi: update API
 .mode_valid()

API changed since:
12c683e12cd8 ("drm: bridge: Pass drm_display_info to drm_bridge_funcs .mode_valid()")

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
---
 drivers/gpu/drm/bridge/cadence/cdns-dp-core.c   | 1 +
 drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/bridge/cadence/cdns-dp-core.c b/drivers/gpu/drm/bridge/cadence/cdns-dp-core.c
index c059d56b4f46..cb4897c664f0 100644
--- a/drivers/gpu/drm/bridge/cadence/cdns-dp-core.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-dp-core.c
@@ -256,6 +256,7 @@ static int cdns_dp_bridge_attach(struct drm_bridge *bridge,
 
 static enum drm_mode_status
 cdns_dp_bridge_mode_valid(struct drm_bridge *bridge,
+			const struct drm_display_info *info,
 			  const struct drm_display_mode *mode)
 {
 	enum drm_mode_status mode_status = MODE_OK;
diff --git a/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c b/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
index 5f2442fa761f..1e5130e295f7 100644
--- a/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
+++ b/drivers/gpu/drm/bridge/cadence/cdns-hdmi-core.c
@@ -359,6 +359,7 @@ static int cdns_hdmi_bridge_attach(struct drm_bridge *bridge,
 
 static enum drm_mode_status
 cdns_hdmi_bridge_mode_valid(struct drm_bridge *bridge,
+			    const struct drm_display_info *info,
 			  const struct drm_display_mode *mode)
 {
 	struct cdns_mhdp_device *mhdp = bridge->driver_private;
-- 
2.29.2

